sdk/src/main/java/com/google/cloud/dataflow/sdk/io/BigQueryIO.java

```diff
@@ -1757,6 +1800,28 @@ public Bound to(
       }
 
       /**
+       * Returns a copy of this write transformation, but writing to the specified table. Refer to
+       * {@link #parseTableSpec(String)} for the specification format.
+       *
+       * <p>Does not modify this object.
+       */
+      public Bound to(ValueProvider<String> tableSpec) {
+        return toTableRef(NestedValueProvider.of(tableSpec, new TableSpecToTableRef()));
+      }
+
+      /**
+       * Returns a copy of this write transformation, but writing to the specified table.
+       *
+       * <p>Does not modify this object.
+       */
+      private Bound toTableRef(ValueProvider<TableReference> table) {
```

Comment:
I don't understand this change. It's just the VP version of `to(TableReference table)`, right? why change the name.